Function/Function.csproj

```diff
@@ -7,12 +7,14 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.Data.Tables" Version="12.8.2" />

```

Comment:
Here I am adding the two packages required to interact with the table storage client. `Azure.Data.Tables` is the table storage client itself and `Microsoft.Extensions.Azure` is so the client can be used with dependency injection in `Program.cs`